src/main/java/net/darkmorford/jas/configuration/ConfigurationData.java

```diff
@@ -0,0 +1,15 @@
+package net.darkmorford.jas.configuration;
+
+public class ConfigurationData {
+	public static final int SNAD_SPEED_INCREASE_DEFAULT = 2;
+	public static final String SNAD_SPEED_INCREASE_KEY = "Snad Speed increase multilpier";
```

Comment:
"multiplier" is misspelled in this string, and the capitalization is inconsistent with the string for Soul Snad.

Reply:
Whoops fixing